package exceptionhandling;

public class arithmetic_unchecked {

	public static void main(String[] args) {
		// TODO Auto-generated method stub

		try {
		int j = 8;
		int k = 0;
		int result;
		result = j/k;
		}
		catch (Exception e)
		{
			
		int a = 10;
		int b = 30;
				System.out.println("addition" + (a+b));
		}
		
	}

}
